### ISSUE TITLE
search maxResults not working

### ISSUE THREAD

**Issue by daakubotto** [Kylart/MalScraper#21]
Although I'm really enjoying this API for my little project there is a tiny issue that causes a lot of stuff to go slower,

in the search function the option maxResults isn't functional. No matter what is put there (1, 250, 40) it will always return 50.

```
const malScraper = require('mal-scraper')
const search = malScraper.search

var anime = search.search('anime', {
    maxResults: 100,
    term: argument
 })
    .catch(console.error)
    .then((results) => console.log(results.length))
```

always returns 50. no matter what you set `maxResults` to.

**Comment by Kylart** [comment]
I am releasing a new version in a few minutes with this bug fixed.

Thanks @daakubotto !

**Comment by crazynike94** [comment]
It don't work again :(

**Comment by JustalK** [comment]
I have fixed the issue and describe the problem inside the pull request

**Comment by Kylart** [comment]
This is now fixed and deployed in v2.11.0!

Thanks @JustalK !
